engine/flink/avro-components-utils/src/test/scala/pl/touk/nussknacker/engine/avro/AvroSchemaSpelExpressionSpec.scala

```diff
@@ -179,14 +169,12 @@ class AvroSchemaSpelExpressionSpec extends FunSpec with Matchers {
     val schema = wrapWithRecordSchema(
       """[
         |  { "name": "stringField", "type": "string" },
-        |  { "name": "mapField", "type": { "type": "map", "values": "string" } },
-        |  { "name": "enumField", "type": { "type": "enum", "name": "sampleEnum", "symbols": ["One", "Two"] } }
+        |  { "name": "mapField", "type": { "type": "map", "values": "string" } }
```

Comment:
Why you removed enum from this test? It was to hard to prepare good test data after changes in enum handling?

Reply:
Because enum isn't string. I've changed it from union [EnumSymbol, String] to just EnumSymbol - before it was done only for allow to pass enum in sink where field is string. Right now validation can handle it and we don't have to do hacks with union.